listener.go

```diff
@@ -67,6 +67,17 @@ func (l *listener) Accept() (net.Conn, error) {
 	return Server(c, l.config)
 }
 
+// AcceptNoHandshake returns a connection that has not completed handshake
+// yet. Call Server on it to do so.
+func (l *listener) AcceptNoHandshake() (net.Conn, error) {
+	return l.parent.Accept()
+}
+
+// GetConfig returns the listener config. Intended for use with Server.
+func (l *listener) GetConfig() *Config {
+	return l.config
+}
```

Comment:
Is the getter required here? We're passing in the config to the listener, so we've got it available in mp-relay to also pass into `Server`.

Reply:
It's probably not required indeed, will remove.

Reply:
Done e78f28d